### ISSUE TITLE
Package github.com/friendsofgo/errors checksum mismatch

### ISSUE THREAD

**Issue by mauleyzaola** [volatiletech/sqlboiler#626]
If you're having a generation problem please answer these questions before submitting your issue. Thanks!

### What version of SQLBoiler are you using (`sqlboiler --version`)?
```
SQLBoiler v3.6.0
```

### What is your database and version (eg. Postgresql 10)
```
mysql 5.6.42
```

### If this happened at generation time what was the full SQLBoiler command you used to generate your models? (if not applicable leave blank)
```
mau@MacPro:server$ sqlboiler --wipe --no-tests mysql
```
No errors whatsoever. Then I moved to the `models/` package where the generated code is
```
mau@MacPro:server$ cd models/
mau@MacPro:models$ go build
go: downloading github.com/friendsofgo/errors v0.9.1
verifying github.com/friendsofgo/errors@v0.9.1: checksum mismatch
	downloaded: h1:ksGIdtObXKKxxMq9VgsjKecdLCOlXwsE6XypldOuDi0=
	sum.golang.org: h1:VnyCwEgi1BgGL4wAfCxBOLsSNyJ+iAwJOxh4Gbq7aI4=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```
I am using go modules 

### Please provide a relevant database schema so we can replicate your issue (Provided you are comfortable sharing this)
I cannot share the db schema because there is a NDA going on. However I think the error is not related to the schema at all

### Further information. What did you do, what did you expect?
I just wanted to test the generated code was building fine.
Expected no errors when building

**Comment by aarondl** [comment]
This is not something we can help with as sqlboiler does not use modules and therefore does not maintain a go.sum at all. You'll have to figure out what's going on locally on your machine. Maybe the friendsofgo/errors package force pushed something?

**Comment by aarondl** [comment]
See https://github.com/friendsofgo/errors/issues/2

**Comment by aperezg** [comment]
So, we fix that already, we can use the new v0.9.2, sorry for the inconveniences

**Comment by mauleyzaola** [comment]
Version ` v0.9.2` works fine thank you guys
